packages/material-renderers/src/complex/DeleteDialog.tsx

```diff
@@ -37,13 +37,17 @@ export interface DeleteDialogProps {
     onClose(): void;
     onConfirm(): void;
     onCancel(): void;
+    headerText: string,
```

Comment:
I think title describes more precisely what this is used for. Please rename to `title`